src/classes/game.py

```diff
@@ -0,0 +1,39 @@
+"""Module that implements Game class."""
+import datetime
+
+
+class Game:
+    """Class that implements game."""
+
+    id = int
```

Comment:
Are you sure is should be like this `id = int` instead of this `id: int`

Reply:
Not sure, ok, will add annotations

Reply:
Done